### PR TITLE
Add top-level README and some basic CircleCI integration.

### PR DESCRIPTION
**Pull request by rfk**
This adds some motivational stuff to the readme, which I've moved into the top level of the repo and fleshed out a bit.  I also wanted to copy across some of the publishing automation from the application-services repo, but I don't think I can do that without admins rights on this repo (ref https://github.com/mozilla/ecosystem-platform/issues/3).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: 2.1
+
+jobs:
+  Deploy to GitHub pages:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "f1:3e:05:3a:66:69:60:4f:1f:9f:8f:af:6e:1f:0f:53"
```
Comment (rfk) [inline .circleci/config.yml]: @vladikoff did you help set this up on the app-services repo?

Comment (vladikoff) [reply .circleci/config.yml]: @eoger I think?

Comment (eoger) [reply .circleci/config.yml]: This was @linacambridge actually

Comment (linacambridge) [reply .circleci/config.yml]: Yep, that was me! I'll weigh in on this after my 1:1 with @rfk. 😆

**Comment by linacambridge** [comment]
This was surprisingly easy to set up on a-s! I added a [read-write deploy key](https://github.com/mozilla/application-services/settings/keys) for the repo, then cribbed the changes to `config.yml` from the [Docusaurus docs](https://docusaurus.io/docs/en/publishing). Adding the deploy key needs admin rights, but, once that's done, the rest of this PR looks great!

**Comment by linacambridge** [comment]
This should be good to go now! 🚀 The private key is in [CircleCI](https://circleci.com/gh/mozilla/ecosystem-platform/edit#ssh) and the public key is in the [deploy keys for this repo](https://github.com/mozilla/ecosystem-platform/settings/keys).

**Pull request merged**